Array10toSquaredArray/Array10toSquaredArray/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace Array10toSquaredArray
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
```

Comment:
Why you create 2 arrays?
Based on issue we should create 1 array and fill him by squared indexes. Seems like it can be solved by 1 array.